userguide/dailyuse/anbox.rst

```diff
@@ -1,13 +1,15 @@
 Android apps
 ========================
 
-`Anbox <https://anbox.io>`_ is a minimal Android container and compatibility layer that allows you to run Android apps on GNU/Linux operating systems such as UBports.
+`Anbox <https://anbox.io>`_ is a minimal Android container and compatibility layer to run Android apps on GNU/Linux operating systems such as UBports.
 
 .. note::
-    When "host" is used in this document, it refers to another device which you can connect your Ubuntu Touch device to. Your host device must have ``adb`` and ``fastboot`` installed.
+    "Computer" refers to another device you can connect your Ubuntu Touch device to.
```

Comment:
We can also make it more practical. It's not so much whether you **can** connect it. It's the device that actually **will** connect.

"Computer" refers to the device you connect to your Ubuntu Touch device.

Also please use the same verb here and in the next sentence. connect-connected or attach-attached, but no mix

Reply:
```suggestion
    "Computer" refers to another device you connect your Ubuntu Touch device to.
```